### DISCUSSION TITLE
How practical is it to have different layouts like capacities ?

### DISCUSSION THREAD

**Discussion by arromattic** [TriliumNext/Trilium#4070] [Ideas]
capacities has four different layout 
1. Wiki
wiki like page 
2. person or biography 
for people
3. page
normal page
4. index cards
a simple card

**Comment by XXXJetfireXXX** [comment]
Not sure I understand...
You can do any of these using text notes. There are a lot of tools at your disposal, like text formatting, image insertion, tables, etc...

**Comment by arromattic** [comment]
I meant this 
https://docs.capacities.io/reference/page-layouts

**Comment by zadam** [comment]
Something kind of similar are [templates](https://github.com/zadam/trilium/wiki/Template) which allow you to define some default structure of the note. With tables and/or promoted attributes you can kind of create a specific layout too.
